lib.php

```diff
@@ -775,55 +775,41 @@ function surveypro_extend_settings_navigation(settings_navigation $settings, nav
         return;
     }
 
-    $paramurl = ['s' => $cm->instance];
-
-    // First tab; "Surveypro".
-    $condition = surveypro_get_link_visibility_condition('surveypro');
+    // Surveypro.
+    list($condition, $label, $url) = surveypro_get_link_visibility_condition('surveypro');
```

Comment:
I imagine that you're aware that,[ since PHP 7.1 (from memory)](https://stitcher.io/blog/array-destructuring-with-list-in-php), you can write the above as:

```suggestion
    [$condition, $label, $url] = surveypro_get_link_visibility_condition('surveypro');
```

Reply:
No, I did not! I will push a new PR to renew the old syntax in all over the code. Thanks.